sql/npc_list.sql

```diff
@@ -9010,8 +9010,8 @@ INSERT INTO `npc_list` VALUES (17122288,'Shanene','Shanene',0,161.183,0.468,91.1
 INSERT INTO `npc_list` VALUES (17122289,'Survival_Guide','Survival Guide',192,-404.000,-16.000,-163.000,1,50,50,0,17,32,0,3,0x0000810900000000000000000000000000000000,0,'SURVIVAL_GUIDE',1);
 -- NC: INSERT INTO `npc_list` VALUES (17122290,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
 -- NC: INSERT INTO `npc_list` VALUES (17122291,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
--- NC: INSERT INTO `npc_list` VALUES (17122292,'NOT_CAPTURED','???',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
-
+INSERT INTO `npc_list` VALUES (17122292,'qm7','???',129,-44.530,0.127,431.877,1,50,50,0,0,0,0,3,0x0000780900000000000000000000000000000000,0,NULL,0);
+-- https://drive.google.com/drive/folders/1ZFOrZUGQr1G4EJ89fZAn9hEKNe05w9bk
```

Comment:
One last thing, and I apologize for not calling this out earlier.  Could you please remove the references from the sql file, and just put them in the PR?  Trying to keep things tidy in the file itself.

Reply:
sure thing, still not sure what is wanted in file and what is better in pr

Reply:
Understood.  For NPCs there's a fair amount of automation, and much less manual references outside of some notes here and there.  This is very different from some other sql files such as synth and mods.